engine_test.go

```diff
@@ -103,3 +103,35 @@ func TestWithRequestContentType(t *testing.T) {
 		require.False(t, ok)
 	})
 }
+
+func TestWithResponseContentType(t *testing.T) {
+	t.Run("base", func(t *testing.T) {
+		e := NewEngine()
+		require.NotNil(t, e.responseContentTypes)
+		assert.Equal(t, []string{"application/json", "application/xml"}, e.responseContentTypes)
```

Comment:
I'm unsure you need the require line


```suggestion
		assert.NotNil(t, e.responseContentTypes)
		assert.Equal(t, []string{"application/json", "application/xml"}, e.responseContentTypes)
```

Or maybe this,

```suggestion
		assert.NotEmpty(t, e.responseContentTypes)
		assert.Equal(t, []string{"application/json", "application/xml"}, e.responseContentTypes)
```

But I don't think you need it


```suggestion
		assert.Equal(t, []string{"application/json", "application/xml"}, e.responseContentTypes)
```